intake_esm/aggregate.py

```diff
@@ -103,20 +95,23 @@ def concat_time_levels(dsets):
     dsets : list
         A list of datasets to concatenate.
 
+    time_coord_name_default : string
+        Default name of the time coordinate
+
     Returns
     -------
     dset : xarray.Dataset,
         The concatenated dataset.
     """
-
+    dsets = dask.compute(*dsets)
```

Comment:
Does this belong here or up a level in the module that is calling `open_dataset`? It seems like it would at least be prudent to check if `dsets` actually contains `dask.delayed` objects.